packages/Manager/src/main/resources/db/migration/V0000__setup_postgis.sql

```diff
@@ -1,2 +0,0 @@
-CREATE EXTENSION
-  IF NOT EXISTS postgis;
```

Comment:
This needs to be run as a super user, so with these changes this now needs to be run as a manual step when setting up a new RDS instance